implementations/rust/ockam/ockam_identity/src/secure_channel/handshake/mod.rs

```diff
@@ -0,0 +1,11 @@
+mod error;
+
+/// This directive makes sure that we only run the handshake protocol if it has been compiled
+/// on a little endian system since it is not supporting a big endian one at the moment
+#[cfg(target_endian = "little")]
```

Comment:
I think it could be a descriptive compilation error instead of a "not found" error we'll get because this module just won't be included during the compilation. But that's fine if you don't want to spend time fixing it

Reply:
Yes I agree this is not great. Is there an easy way to put a custom compilation error message in that case?

Reply:
Sure,
```
#[cfg(not(target_endian = "little"))]
compile_error!("Key Exchange is only supported on little-endian machines");
```